File: javascript/easy/how_many_numbers_are_smaller.js
// 1365. How Many Numbers Are Smaller Than the Current Number
// https://leetcode.com/problems/how-many-numbers-are-smaller-than-the-current-number/

var smallerNumbersThanCurrent = function(nums) {
    var answer = [];
    for (var i = 0; i < nums.length; i++) {
        var count = 0;
        for (var j = 0; j < nums.length; j++) {
            if (nums[i] > nums[j]) {
                count++;
            }
        }
        answer.push(count);
    }
    return answer;
};

File: python/easy/how_many_numbers_are_smaller.py
# 1365. How Many Numbers Are Smaller Than the Current Number
# https://leetcode.com/problems/how-many-numbers-are-smaller-than-the-current-number/

class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        ans = []
        for i in range(len(nums)):
            count = 0
            for j in range(len(nums)):
                if nums[i] > nums[j]:
                    count = count + 1
            ans.append(count)
        
        return ans